src/PIL/BmpImagePlugin.py

```diff
@@ -347,6 +347,6 @@ def _save(im, fp, filename):
 Image.register_open(BmpImageFile.format, BmpImageFile, _accept)
 Image.register_save(BmpImageFile.format, _save)
 
-Image.register_extension(BmpImageFile.format, ".bmp")
+Image.register_extensions(BmpImageFile.format, [".bmp", ".dib"])
```

Comment:
https://www.imagemagick.org/script/formats.php

> DIB is a BMP file without the BMP header.

Since the BMP header is used to accept files for BmpImageFile, adding support is not as simple here as adding another extension.

Reply:
Ok, removed that and created new issue.
#3689